SU-SWS/stanford_publication
# D8CORE-3536 Add pseudo field for easily displaying the citation type

# READY FOR REVIEW

# Summary
- Add pseudo field for easily displaying the citation type

# Need Review By (Date)
- 3/24

# Urgency
- medium

# Steps to Test
1. follow steps in https://github.com/SU-SWS/stanford_profile/pull/387
2. after creating the basic page, verify while logged out that the citation type is displayed at the top of the card

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
